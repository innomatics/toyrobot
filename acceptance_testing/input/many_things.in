Any lines without recognised commands will be ingored (line this one)

Blank lines also ignored.

Checking that COMMANDS ingored until a PLACE occurs
REPORT
MOVE
REPORT
LEFT
MOVE

None of that so far should do anything.
OK now lets place the bot
PLACE 4,4,WEST
REPORT
now lets place again
PLACE 2,3,NORTH
REPORT
