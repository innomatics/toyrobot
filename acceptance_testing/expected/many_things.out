4,4,WEST
2,3,NORTH
